albumentations/core/composition.py

```diff
@@ -170,10 +171,25 @@ def __init__(self, transforms, bbox_params=None, keypoint_params=None, additiona
 
         self.add_targets(additional_targets)
 
+        self.is_check_args = True
```

Comment:
Please add this var into `_to_dict`.
And reformat code with `black` https://albumentations.ai/docs/contributing/